Designs/UI /signup.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+    <head>
+           <link rel="stylesheet" href="css/style.css">
+            <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css">
+    </head>
+<body>
+        <div class="container">
+                <h2>Sign up to register</h2>
+                <form action="#">
+                  <div class="form-group">
+                    <label for="name">First Name:</label>
+                    <input type="name" class="form-control" id="name" placeholder="Enter First Name" name="name">
+                    <label for="name">Last Name:</label>
+                    <input type="name" class="form-control" id="name" placeholder="Enter Last Name" name="name">
+                    <label for="name">User Name:</label>
+                    <input type="name" class="form-control" id="name" placeholder="Enter User Name" name="name">
+                    <label for="email">Email:</label>
+                    <input type="email" class="form-control" id="email" placeholder="Enter email" name="email">
+                  </div>
+                  <div class="form-group">
+                    <label for="pwd">Password:</label>
+                    <input type="password" class="form-control" id="pwd" placeholder="Enter password" name="pwd">
+                    <label for="pwd">Confirm Password:</label>
+                    <input type="password" class="form-control" id="pwd" placeholder="Re-Enter password" name="pwd">
+                  </div>
+                  <div class="checkbox">
+                    <label><input type="checkbox" name="remember"> Remember me</label>
+                  </div>
+                  <button type="button" class="btn btn-default">Sign Up</button>
+                   <a href="User-login.html">Already member? Click here to login</a>
+                </form>
+              </div>
+</body>
```

Comment:
Add **an** extra line at the end of your file

Reply:
noted thanx